KarhooUISDK/Screens/QuoteList/QuoteListViewController.swift

```diff
@@ -30,24 +30,49 @@ final class KarhooQuoteListViewController: UIViewController, BaseViewController,
     private lazy var tableHeaderStackView = UIStackView().then {
         $0.translatesAutoresizingMaskIntoConstraints = false
         $0.axis = .vertical
-        $0.distribution = .fill
-        $0.alignment = .fill
+        $0.alignment = .center
         $0.spacing = UIConstants.Spacing.standard
     }
     private var headerViews: [UIView] {
-        [
+        #if DEBUG
+        return [
+            addressPickerView,
+            buttonsStackView,
```

Comment:
why we need to return different objects for debug/production? Next uisdk version will be released when sorting and filtering will be implemented.